packages/core/admin/_internal/node/develop.ts

```diff
@@ -2,17 +2,20 @@ import type { CLIContext } from '@strapi/strapi';
 import * as tsUtils from '@strapi/typescript-utils';
 import { joinBy } from '@strapi/utils';
 import chokidar from 'chokidar';
+import * as fs from 'fs-extra';
```

Comment:
Small nit — do we have to use fs-extra? I thought we were wanting to phase it out since we can use "node:fs/promises"

Reply:
It was a copy/paste of the code that was there before the build changes, I didn't want to rewrite it, but I could

Reply:
Done